esphome/components/xiaomi_ble/xiaomi_ble.cpp

```diff
@@ -69,40 +97,60 @@ optional<XiaomiParseResult> parse_xiaomi(const esp32_ble_tracker::ESPBTDevice &d
     return {};
   }
 
-  if (!device.get_service_data_uuid()->contains(0x95, 0xFE)) {
+  if (!device.get_service_data_uuid()->contains(0x95, 0xFE) and !device.get_service_data_uuid()->contains(0x1D, 0x18) and !device.get_service_data_uuid()->contains(0x1B, 0x18))) {
```

Comment:
```suggestion
  if (!device.get_service_data_uuid()->contains(0x95, 0xFE) && !device.get_service_data_uuid()->contains(0x1D, 0x18) && !device.get_service_data_uuid()->contains(0x1B, 0x18))) {
```

See also linter output, this line is too long

Reply:
OK.